code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm

```diff
@@ -453,3 +453,59 @@
 	icon_state = "gruboid_tentacle_wiggle"
 	wiggle = "gruboid_tentacle_spawn"
 	retract = "gruboid_tentacle_retract"
+
+//Whitesands Goliath
+/mob/living/simple_animal/hostile/asteroid/goliath/beast/whitesands
+	name = "goliath"
+	desc = "A species of goliath native to sand planets. While its shell can take more punishment, its also has much weaker skin to compensate"
+	icon = 'icons/mob/lavaland/lavaland_monsters_wide.dmi'
+	icon_state = "ws_goliath"
+	icon_living = "ws_goliath"
+	icon_aggro = "ws_goliath"
+	icon_dead = "ws_goliath_dead"
+	throw_message = "does nothing to the tough hide of the"
+	pre_attack_icon = "ws_goliath_preattack"
+
+	move_to_delay = 2.5 SECONDS
+	speed = 2
+
+	maxHealth = 30
+	health = 30
+	armor = list("melee" = 65, "bullet" = 75, "laser" = 45, "energy" = 20, "bomb" = 50, "bio" = 30, "rad" = 30, "fire" = 30, "acid" = 30)
```

Comment:
Way too much. I'd just make them goliath reskins honestly or make them work like basilisks. They're literally immune to most types of guns.

Reply:
reduced by like 45%